x-pack/plugins/security_solution/public/management/pages/response_actions/view/response_actions_list.tsx

```diff
@@ -0,0 +1,475 @@
+            const status = data.isExpired
+              ? UX_MESSAGES.badge.failed
+              : isCompleted
+              ? data.wasSuccessful
+                ? UX_MESSAGES.badge.completed
+                : UX_MESSAGES.badge.failed
+              : UX_MESSAGES.badge.pending;
+
+            return (
+              <EuiToolTip content={status} anchorClassName="eui-textTruncate">
+                <EuiBadge
+                  data-test-subj={getTestId('column-status')}
+                  color={
+                    data.isExpired
+                      ? 'danger'
+                      : isCompleted
+                      ? data.wasSuccessful
+                        ? 'success'
+                        : 'danger'
+                      : 'warning'
+                  }
+                >
+                  <FormattedMessage
+                    id="xpack.securitySolution.responseActionsList.list.item.status"
+                    defaultMessage="{status}"
+                    values={{ status }}
+                  />
+                </EuiBadge>
+              </EuiToolTip>
+            );
+          },
+        },
+        {
+          field: '',
+          align: RIGHT_ALIGNMENT as HorizontalAlignment,
+          width: '40px',
+          isExpander: true,
+          name: (
+            <EuiScreenReaderOnly>
+              <span>{UX_MESSAGES.screenReaderExpand}</span>
+            </EuiScreenReaderOnly>
+          ),
+          render: (data: ActionDetails) => {
+            return (
+              <EuiButtonIcon
+                onClick={() => toggleDetails(data)}
+                aria-label={itemIdToExpandedRowMap[data.id] ? 'Collapse' : 'Expand'}
+                iconType={itemIdToExpandedRowMap[data.id] ? 'arrowUp' : 'arrowDown'}
+              />
+            );
+          },
+        },
+      ];
+      if (hideHostNameColumn) {
+        columns.splice(3, 1);
+      }
+      return columns;
+    }, [getTestId, hideHostNameColumn, itemIdToExpandedRowMap, toggleDetails]);
+
+    // table pagination
+    const tablePagination = useMemo(() => {
+      return {
+        pageIndex: queryParams.page || 0,
+        pageSize: queryParams.pageSize || 10,
+        totalItemCount: totalItemCount || 0,
+        pageSizeOptions: MANAGEMENT_PAGE_SIZE_OPTIONS as number[],
+      };
+    }, [queryParams, totalItemCount]);
+
+    // handle onChange
+    const handleTableOnChange = useCallback(
+      ({ page: _page }: CriteriaWithPagination<ActionDetails>) => {
+        const { index, size } = _page;
+        setQueryParams((prevState) => ({
+          ...prevState,
+          page: index,
+          pageSize: size,
+        }));
+        reFetchEndpointActionList();
+      },
+      [reFetchEndpointActionList, setQueryParams]
+    );
+
+    return (
+      <AdministrationListPage
+        data-test-subj="responseActionsPage"
+        title={hideHeader ? undefined : UX_MESSAGES.pageTitle}
+      >
+        {isFetched && !totalItemCount ? (
```

Comment:
What if there is no result already? Do we want to show this message about changing filters?
I think we should do the same we do for artifacts list page, check if there is any and then if the filters match results or not.

What about the date range input? I don't see it in the code but it appears in the screenshots.

Reply:
Yeah, this is temp text for now. There's no UX for empty results. I'll ask @merilmathew about it! Thanks for pointing it out.

I'll push the date range inputs a bit later. Testing it locally to make sure it works correctly

Reply:
updated screenshots with the date filter